mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -54,6 +60,33 @@ def _find_nan_change(before_image, filtered_image_data):
     return nan_change
 
 
+def sub(x: Tuple[int, int]) -> int:
+    """
+    Subtracts two tuples. Helper method for generating the negative slice list.
+    :param x: The int tuple.
+    :return: The first value subtracted from the second value.
+    """
+    return x[1] - x[0]
+
+
+def _generate_slices_range_list(slices: List[int]) -> List[str]:
```

Comment:
Could you rename this to _group_consecutive_values() to make it clearer what it does.

And also add a simple unit test just for this function.